### PR TITLE
SCMOD-10189: Updated to add tomcat-juli module for access logs

### PR DESCRIPTION
**Pull request by thilagasanthosh**
https://portal.digitalsafe.net/browse/SCMOD-10189

### CODE REVIEW

File: release-notes-3.0.0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+${version-number}
+
+#### New Features
+
+ - **SCMOD-10189** : The new caf-logging-tomcat-juli module added to support standard logging format mentioned in the [caf-logging](https://github.com/CAFapi/caf-logging#example) for access logs 
+
+#### Breaking changes
+
+- **SCMOD-10189** : The new caf-logging-tomcat-juli module added to support standard logging format mentioned in the [caf-logging](https://github.com/CAFapi/caf-logging#example) for access logs.
+ To avoid logging related class loading problems due to JRE11 changes in tomcat, the `slf4j-api` dependency scope must be changed to `provided` in a module where healthcheck api is implemented or called. Example slf4j-api scope should be made provided in [common-ui-health](https://github.houston.softwaregrp.net/Verity/common-lib/tree/master/common-ui-health).
```
Comment (michael-bryson) [inline release-notes-3.0.0.md:13]: You can't put a link to the internal github here, this is open source repository so non-MF people will not be able to access that link.

File: release-notes-3.0.0.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+This healthcheck api is called from ECA UI, Data management UI, and admin UI.
```
Comment (michael-bryson) [inline release-notes-3.0.0.md:14]: Again this is an open source repository so don't mention internal usage of this image in our applications as they are meaningless to someone outside our product team.

**Comment by thilagasanthosh** [comment]
> These look like good changes but I'm confused about the move to `3.0.0-SNAPSHOT`. Didn't you say earlier that these changes meant that there is no need to update the downstream images? If so then there is no breaking change.

Since this was draft PR, I was about to revert this change. But I was verifying some issues in documentid-service side logs. I will revert it now.

**Comment by thilagasanthosh** [comment]
The services documentid-service, admin-ui, datamanagement-ui, tag-service, caf-storage, audit-service are updated with the latest tomcat image. The deployments were fine on larry.

The job-service and eca-ui are pending. After completing the testing, will update the status here.

The job-service dev build is failing due to artifactory issue.


--

**Pull request merged**